apis/capabilities/v1beta1/application_types.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2020 Red Hat.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"github.com/3scale/3scale-operator/pkg/common"
+	"github.com/go-logr/logr"
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"reflect"
+)
+
+const (
+	ApplicationReadyConditionType common.ConditionType = "Ready"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ApplicationSpec defines the desired state of Application
+type ApplicationSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	//AccountCRName name of account custom resource under which the application will be created
+	AccountCRName *corev1.LocalObjectReference `json:"accountCRName"`
```

Comment:
@Patryk-Stefanski 
Just starting to look at this today
Just wondering if the application CR's in the verification steps are correct
```
kubectl apply -f - <<EOF
---
apiVersion: capabilities.3scale.net/v1beta1
kind: Application
metadata:
  name: example
  namespace: 3scale-test
spec:
  accountCRName: developeraccount01
  applicationPlanName: plan02
  productCRName: product1-cr
  name: testApp12 
  description: further testing12
  suspend: true
EOF ```
because you are using LocalObjectReference as a type
accountCRName, productCRName would be something like
```bash
  accountCRName: 
    name: test
  productCRName:
    name: test
```

Reply:
Yea it looks like verification steps need updating
![image](https://user-images.githubusercontent.com/16667688/189328923-de76762c-fe06-4e70-8820-11e9917e7da1.png)